src/runner/func_runner.cpp

```diff
@@ -8,46 +8,19 @@
 #include <faabric/util/logging.h>
 #include <faabric/util/timing.h>
 #include <faaslet/Faaslet.h>
+#include <runner/runner_utils.h>
 #include <storage/FileLoader.h>
 #include <wasm/WasmModule.h>
 
-#include <boost/program_options.hpp>
-#include <iostream>
-
 namespace po = boost::program_options;
 
 int doRunner(int argc, char* argv[])
 {
     faabric::util::initLogging();
 
-    // Define command line arguments
```

Comment:
Moved this to `runner_utils.cpp` to be used both by `func_runner` and `local_pool_runner`.